components/dashboard/src/menu/OrganizationSelector.tsx

```diff
@@ -60,11 +60,11 @@ export default function OrganizationSelector() {
     if (currentOrg.data) {
         if (repoConfigListAndDetail) {
             linkEntries.push({
-                title: "Repositories",
-                customContent: <LinkEntry>Repositories</LinkEntry>,
+                title: "Configurations",
```

Comment:
I didn't think we wanted to call this Configurations in the UI and we're leaving it as Repositories. Same w/ the actual URL - `/repositories`